vulkan/vulkan_handles.hpp

```diff
@@ -2888,6 +2848,7 @@ namespace VULKAN_HPP_NAMESPACE
   public:
     using CType      = VkDisplayKHR;
     using NativeType = VkDisplayKHR;
+    using ParentType = NoParent;
```

Comment:
Shouldn't `PhysicalDevice` be the parent type?

Reply:
it is automatically generated